src/ignitus-CareersPage/styles.ts

```diff
@@ -0,0 +1,72 @@
+import styled from '@emotion/styled';
+import * as C from '../ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/colors';
+import * as F from '../ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/fonts';
+import { StyleTypes} from './types';
+import {Heading4} from '../ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/typography';
+import {minimumWidthQuery} from '../ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/media';
+
+export const Container = styled.div`
+  background-color: ${C.White};
```

Comment:
Remove this background color property, and leave to grey, that is present in other screens.

Reply:
okay